### PR TITLE
Update to API 4.6

### PR DESCRIPTION
**Pull request by dragneelfps**
Related to Issue https://github.com/elbekD/kt-telegram-bot/issues/18

### CODE REVIEW

File: library/src/main/kotlin/com/elbekD/bot/TelegramBot.kt [hunk 1]
```diff
@@ -1,29 +1,7 @@
...
-import com.elbekD.bot.types.MaskPosition
-import com.elbekD.bot.types.Message
-import com.elbekD.bot.types.PassportElementError
-import com.elbekD.bot.types.Poll
-import com.elbekD.bot.types.PreCheckoutQuery
-import com.elbekD.bot.types.ReplyKeyboard
-import com.elbekD.bot.types.ShippingOption
-import com.elbekD.bot.types.ShippingQuery
-import com.elbekD.bot.types.Update
+import com.elbekD.bot.types.*
```
Comment (elbekD) [inline library/src/main/kotlin/com/elbekD/bot/TelegramBot.kt]: do not use * imports

File: library/src/main/kotlin/com/elbekD/bot/TelegramBot.kt [hunk 2]
```diff
@@ -589,7 +567,8 @@ internal abstract class TelegramBot protected constructor(username: String, tk:
                                        errors: List<PassportElementError>) =
             client.setPassportDataErrors(userId, errors)
 
-    override fun sendPoll(chatId: Any, question: String, options: List<String>, disableNotification: Boolean?, replyTo: Int?, markup: ReplyKeyboard?) =
+    override fun sendPoll(chatId: Any, question: String, options: List<String>, disableNotification: Boolean?, replyTo: Int?, markup: ReplyKeyboard?,
+                          anonymous: Boolean?, type: String?, allowsMultipleAnswers: Boolean?, correctOptionId: Int?, closed: Boolean?) =
             client.sendPoll(chatId, question, options, disableNotification, replyTo, markup)
```
Comment (elbekD) [inline library/src/main/kotlin/com/elbekD/bot/TelegramBot.kt]: did u miss adding new params to client.sendPoll method?

File: library/src/main/kotlin/com/elbekD/bot/http/TelegramApi.kt [hunk 3]
```diff
@@ -1,25 +1,6 @@
...
-import com.elbekD.bot.types.PassportElementError
-import com.elbekD.bot.types.Poll
-import com.elbekD.bot.types.ReplyKeyboard
-import com.elbekD.bot.types.ShippingOption
-import com.elbekD.bot.types.StickerSet
-import com.elbekD.bot.types.Update
-import com.elbekD.bot.types.User
-import com.elbekD.bot.types.UserProfilePhotos
-import com.elbekD.bot.types.WebhookInfo
+import com.elbekD.bot.types.*
```
Comment (elbekD) [inline library/src/main/kotlin/com/elbekD/bot/http/TelegramApi.kt]: imports

File: library/src/main/kotlin/com/elbekD/bot/http/TelegramClient.kt [hunk 4]
```diff
@@ -1,35 +1,11 @@
...
-import com.elbekD.bot.types.Poll
-import com.elbekD.bot.types.ReplyKeyboard
-import com.elbekD.bot.types.ShippingOption
-import com.elbekD.bot.types.StickerSet
-import com.elbekD.bot.types.TelegramObject
-import com.elbekD.bot.types.Update
-import com.elbekD.bot.types.User
-import com.elbekD.bot.types.UserProfilePhotos
-import com.elbekD.bot.types.WebhookInfo
+import com.elbekD.bot.types.*
```
Comment (elbekD) [inline library/src/main/kotlin/com/elbekD/bot/http/TelegramClient.kt]: imports

File: library/src/main/kotlin/com/elbekD/bot/http/TelegramClient.kt [hunk 5]
```diff
@@ -1,35 +1,11 @@
...
 import com.elbekD.bot.util.AllowedUpdate
 import com.google.gson.Gson
 import com.google.gson.reflect.TypeToken
-import okhttp3.MediaType
-import okhttp3.MultipartBody
-import okhttp3.OkHttpClient
-import okhttp3.Request
-import okhttp3.RequestBody
-import okhttp3.Response
+import okhttp3.*
```
Comment (elbekD) [inline library/src/main/kotlin/com/elbekD/bot/http/TelegramClient.kt]: imports

File: library/src/test/kotlin/com/elbekD/bot/TelegramBotTest.kt [hunk 6]
```diff
@@ -5,9 +5,7 @@ import com.elbekD.bot.types.InlineKeyboardButton
 import com.elbekD.bot.types.InlineKeyboardMarkup
 import com.elbekD.bot.util.Action
 import com.google.gson.Gson
-import org.junit.Assert.assertEquals
-import org.junit.Assert.assertNotNull
-import org.junit.Assert.assertTrue
+import org.junit.Assert.*
```
Comment (elbekD) [inline library/src/test/kotlin/com/elbekD/bot/TelegramBotTest.kt]: imports

**Review by elbekD — CHANGES_REQUESTED** [review]

**Comment by elbekD** [comment]
Hi @dragneelfps! Are you going to solve comments in a few days?

**Comment by dragneelfps** [comment]
@elbekD you can go ahead and add commits to resolve the comments.

**Pull request merged**